Assets/Editor/BuildScript.cs

```diff
@@ -1,91 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using UnityEditor;
-
-namespace PatchKit.Unity.Editor
-{
-    public class BuildScript
```

Comment:
I'm afraid that deleting this class will break the compatibility with our CI - @genail am I right?

Reply:
Oh I didn't notice it at first. That's true, we can't delete that!

Reply:
@Dzejkop that means we need to bring it back. Can you do it?

Thanks!

Reply:
Sure